packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -15,15 +15,37 @@ export interface Spacing {
   l2: string;
 }
 
+export type ShadowStyle = ShadowStyleIOS & Pick<ViewStyle, 'elevation'>;
+
+export interface ShadowValue {
+  ambient?: ShadowStyle;
+  key: ShadowStyle;
+}
+
+export interface Effects {
+  borderRadiusSmall: number;
```

Comment:
Maybe the border Radius can be its own interface/grouping? I don't know if I would group it under "Effects."

Design has border radius as its own group of global tokens separate from shadows/elevation, so I would probably align with that.

Reply:
Yea, in the Fabric 7 theme object had both of these under theme.effects and that's what I was initially copying but it looks like the new Fluent theme object organizes things like theme.alias.color.neutral, theme.alias.shadow

We don't necessarily have to copy them, but it's worth considering prior art and I'm curious to hear what other folks think